back/engines/commercial/project_folders/app/serializers/project_folders/web_api/v1/notifications/project_folder_moderation_rights_received_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectFolders::WebApi::V1::Notifications::ProjectFolderModerationRightsReceivedSerializer < ::WebApi::V1::Notifications::NotificationSerializer
```

Comment:
There is also `WebApi::V1::Notifications::ProjectModerationRightsReceivedSerializer` in the main app, can we delete this one?